lbz/resource.py

```diff
@@ -84,4 +85,4 @@ def get_guest_authorization(self) -> str:
         It should be overwritten after inheritance with a method to obtain guest auth.
         """
         logger.error("Using default guest authorization which gives root access.")
-        return self._authorizer.sign_authz({"allow": {"*": "*"}, "deny": {}})
+        return self._authorizer.sign_authz({"allow": {"*": "*"}, "deny": {}}, sample_private_key)
```

Comment:
I would consider removing this guest auth, otherwise we'll need to overwrite it in every authorized api. I think it's more a security leak than a convenience tool - what do you think ? 

Especially now, as it will be signed by sample private key, we'll need to set sample public key (makes no sense for dev now).

Reply:
Good catch, I thought this is some test code when doing basically find + replace for the `sign_authz` method. This should definitely not be here